<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt &quot;Gotham Pro&quot;;
background-color: rgb(255, 255, 255);

</string>
  </property>
  <widget class="QLabel" name="label_121">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Длина:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_900">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_104">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ширина:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_902">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_900">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

background:white;
border-radius:20px;
border:2px solid black;
	font: 14pt &quot;Gotham Pro&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_904">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_903">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_106">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>107</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Высота:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
